azurelinuxagent/common/protocol/util.py

```diff
@@ -319,8 +309,7 @@ def clear_protocol(self):
 
     def get_protocol(self, by_file=False):
         """
-        Detect protocol by endpoints, if by_file is True,
```

Comment:
let's remove the parameter as well (by_file)

Reply:
Done

Reply:
there is a call in that uses the parameter in azurelinuxagent/pa/provision/default.py

Reply:
Cleaned up in latest commit